### PR TITLE
changed fragment_explore frome linerLayout to ConstraintLayout issue #141

### PR DESCRIPTION
**Pull request by MarinaTaran**
Change the fragment_explore layout to Constraint layout  issue #141

### CODE REVIEW

File: app/src/main/res/layout/fragment_explore.xml [hunk 1]
```diff
@@ -1,50 +1,60 @@
...
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@+id/tabLayout"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
 
         <androidx.appcompat.widget.Toolbar
             android:id="@+id/toolbar"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
```
Comment (gconnect) [inline app/src/main/res/layout/fragment_explore.xml]: layout_width should be match parent

File: app/src/main/res/layout/fragment_explore.xml [hunk 2]
```diff
@@ -1,50 +1,60 @@
...
+            <!--android:layout_centerHorizontal="true" />-->
 
         </androidx.appcompat.widget.Toolbar>
 
     </com.google.android.material.appbar.AppBarLayout>
+
     <com.google.android.material.tabs.TabLayout
         android:id="@+id/tabLayout"
-        android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
```
Comment (gconnect) [inline app/src/main/res/layout/fragment_explore.xml]: This should remain

File: app/src/main/res/layout/fragment_explore.xml [hunk 3]
```diff
@@ -1,50 +1,60 @@
...
     android:layout_height="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    tools:context=".ui.home.ExploreFragment"
-    android:orientation="vertical">
+    tools:context=".ui.home.ExploreFragment">
+
 
+    <androidx.viewpager.widget.ViewPager
+        android:id="@+id/viewPager"
+        android:layout_width="wrap_content"
```
Comment (gconnect) [inline app/src/main/res/layout/fragment_explore.xml]: layout_width should be match parent

File: app/src/main/res/layout/fragment_explore.xml [hunk 4]
```diff
@@ -1,50 +1,60 @@
...
+        android:layout_height="0dp"
```
Comment (gconnect) [inline app/src/main/res/layout/fragment_explore.xml]: layout height should be match parent

**Comment by gconnect** [comment]
Please check the comments

File: app/src/main/res/layout/fragment_explore.xml (hunk 1 above)
Comment (MarinaTaran) [reply app/src/main/res/layout/fragment_explore.xml]: I got it

**Comment by MarinaTaran** [comment]
@ujjwalagrawal17  yes, of course

**Comment by ujjwalagrawal17** [comment]
Thanks @MarinaTaran

**Pull request merged**